bentoml/bentos.py

```diff
@@ -477,6 +477,118 @@ def containerize(
         return True
 
 
+@inject
+def test_bento_bundle(
+    tag: str,
+    _bento_store: "BentoStore" = Provide[BentoMLContainer.bento_store],
+):
+    """
+    Test given bento tag by creating docker container and running tests on it.
+    """
+    import sys
+    import time
+
+    from bentoml._internal.utils import buildx
+    from bentoml.client.test_client import Endpoint
+    from bentoml.client.test_client import is_equal
+    from bentoml.client.test_client import TestClient
+    from bentoml.client.test_client import get_test_data
+    from bentoml._internal.bento.build_config import TestOptions
+
+    # init
+    bento = _bento_store.get(tag)
+    test_options = TestOptions.from_yaml(
+        os.path.join(bento.path, "tests", "tests.yaml")
+    )
+
+    # setup bento client
+    bento_client = TestClient()
```

Comment:
Interesting...
thanks @aarnphm 
I think it's better to wait with this one, cause to my understanding there should be a bento client support coming soon,
and therefore maybe it will be better using it instead of the `TestClient` I created.
So this one is to be continued...